conf.d/70-security.conf

```diff
@@ -0,0 +1,3 @@
+[options]
+dbfilter=$ODOO_DB_FILTER
+list_db=$ODOO_LIST_DB
```

Comment:
spaces here too.

BTW, if you are naming this file "security", please move `admin_passwd` here, to be consistent, please.